lib/r10k/environment/base.rb

```diff
@@ -36,11 +41,12 @@ def initialize(name, basedir, dirname, options = {})
     @basedir = basedir
     @dirname = dirname
     @options = options
+    @puppetfile_name = options[:puppetfile_name]
 
     @full_path = File.join(@basedir, @dirname)
     @path = Pathname.new(File.join(@basedir, @dirname))
 
-    @puppetfile  = R10K::Puppetfile.new(@full_path)
```

Comment:
What's going on with the assignment in the last parameter position here? Shouldn't we just be passing `@puppetfile_name`?